### PR TITLE
chore(NextSteps): update cta styling and animation

### PR DESCRIPTION
**Pull request by alexcarpenter**
🔍 [Preview Link](https://react-components-git-acnext-steps-animation-hashicorp.vercel.app/components/nextsteps)

---

## Description

Updates NextSteps Cta styling and animation to match what we are doing in StandaloneLink.

https://user-images.githubusercontent.com/825855/197267885-e6ba9796-5203-4237-8852-c52d00b2f76c.mp4

### PR Checklist 🚀

Items in this checklist may not may not apply to your PR, but please consider each item carefully.

- [x] Add Asana and Preview links above.
- [x] Conduct thorough self-review.
- [x] Add or update tests as appropriate.
- [x] Conduct reasonable cross browser testing for both compatibility and responsive behavior (We have a [Sauce Labs](https://app.saucelabs.com/) account for this, if you don't have access, just ask!).
- [x] Conduct reasonable accessibility review (use the [WAS](https://accessible.org/Web-Accessibility-Standards-WAS-2.pdf) as a guide or an [axe browser plugin](https://www.deque.com/axe/) until we establish more formal checks).
- [x] Identify (in the description above) and document (add Asana tasks on [this board](https://app.asana.com/0/1100423001970639/list)) any technical debt that you're aware of, but are not addressing as part of this PR.

### CODE REVIEW

File: packages/next-steps/index.tsx [hunk 1]
```diff
@@ -61,7 +61,31 @@ export default function NextSteps({
                       <p className={s.tileDescription}>{step.description}</p>
                     ) : null}
                     <span className={s.tileCta}>
-                      {step.cta.title} <IconArrowRight16 />
+                      <span className={s.tileCtaLabel}>{step.cta.title}</span>
```
Comment (nandereck) [inline packages/next-steps/index.tsx:64]: 💭 Non-blocking: looks like we're copying this over from `StandaloneLink`. Any thoughts on abstracting this or parts of this out somewhere?

**Review by nandereck — APPROVED** [review]
LGTM 🎯

File: packages/next-steps/index.tsx (hunk 1 above)
Comment (alexcarpenter) [reply packages/next-steps/index.tsx:64]: The tricky part when thinking about abstracting this is the parent element that is triggering the hover animation. In StandaloneLink it is the link, in NextSteps it's the tile. Not to say its not possible but it felt ok to copy paste this for now.

Comment (nandereck) [reply packages/next-steps/index.tsx:64]: _Oh interestin'_. That makes sense!

**Pull request merged**